diff --git a/node_modules/react-native-safe-area-context/common/cpp/react/renderer/components/safeareacontext/RNCSafeAreaViewShadowNode.cpp b/node_modules/react-native-safe-area-context/common/cpp/react/renderer/components/safeareacontext/RNCSafeAreaViewShadowNode.cpp
index 0000000..0000001 100644
--- a/node_modules/react-native-safe-area-context/common/cpp/react/renderer/components/safeareacontext/RNCSafeAreaViewShadowNode.cpp
+++ b/node_modules/react-native-safe-area-context/common/cpp/react/renderer/components/safeareacontext/RNCSafeAreaViewShadowNode.cpp
@@ -16,10 +16,10 @@ void RNCSafeAreaViewShadowNode::layout(LayoutContext layoutContext) {
   auto insets = getConcreteProps().insets;
 
   if (insets.has_value()) {
-    if (insets->top.unit() != Unit::Undefined) {
+    if (insets->top.isDefined()) {
       yogaNode.getStyle().setPadding(yoga::Edge::Top, insets->top);
     }
-    if (insets->bottom.unit() != Unit::Undefined) {
+    if (insets->bottom.isDefined()) {
       yogaNode.getStyle().setPadding(yoga::Edge::Bottom, insets->bottom);
     }
